AcademyResidentInformationApi/V1/Infrastructure/AcademyContext.cs

```diff
@@ -15,15 +15,35 @@ public AcademyContext(DbContextOptions options) : base(options)
 
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
+            // composite primary key for Address table
             modelBuilder.Entity<Address>()
                 .HasKey(address => new
                 {
                     address.ClaimId,
                     address.HouseId
                 });
-        }
 
-        //public DbSet<TelephoneNumber> TelephoneNumbers { get; set; }
-        //Don't think the database hold up-to-date telephone numbers, we may have some for landlords.
+            // composite primary key for Person table
+            modelBuilder.Entity<Person>()
+                .HasKey(person => new
+                {
+                    person.Id,
+                    person.PersonRef
+                });
+
+            // composite foreign key for Address targeting unique columns in the Person table
+            modelBuilder.Entity<Person>()
+                .HasOne(x => x.Address)
```

Comment:
Is there definitely only one address record for each person?

Reply:
I think it's one-to-one.
From what I remember, the Address and Person tables both have `HouseID` and `ClaimID` columns and you can join the tables on these columns e.g. 
```
JOIN hbhousehold ON hbmember.claim_id = hbhousehold.claim_id
		AND hbmember.house_id = hbhousehold.house_id
```
But I will go and confirm this. 👍